utils/vlog/tests/fig28-lutff-pair/lut/golden.model.xml

```diff
@@ -0,0 +1,10 @@
+<models xmlns:xi="http://www.w3.org/2001/XInclude">
+  <model name="LUT4">
+    <input_ports>
+      <port name="I"/>
```

Comment:
Should this have a combinational output thingy?

Reply:
this LUT is modelled with the following code:
```
localparam INIT = 16'h0000;
assign O = INIT[I];
```
There is no combinational connection between `I` and `O`. O think the model is correct.